docs/Limitations.md

```diff
@@ -224,7 +236,7 @@ Privileged support in Kata is essentially different from `runc` containers.
 Kata does support `docker run --privileged` command, but in this case full access
 to the guest VM is provided in addition to some host access.
 
-The container runs with elevated capabilities within the guest and is granted 
+The container runs with elevated capabilities within the guest and is granted
```

Comment:
This change is unrelated and should be dropped.
Otherwise, lgtm.  Thanks @c3d!

Reply:
Emacs with auto-removal of trailing whitespaces. I'll submit it as a separate patch.

Reply:
I wouldn't even bother, or I'd advocate for a CI that checks for those (as the one from the OpenShift enhancement requests).
Otherwise it'll be a never-ending battle.